gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-mqtt5/src/main/resources/plugin.properties

```diff
@@ -4,4 +4,5 @@ version=${project.version}
 description=${project.description}
 class=io.gravitee.plugin.endpoint.mqtt5.Mqtt5EndpointConnectorFactory
 type=endpoint-connector
-features=
\ No newline at end of file
+features=
+icon=images/mqtt.svg
```

Comment:
it does not miss the moreInfo info ?

Reply:
I'm waiting... but maybe we can add in other PR